tests/csm/rest/test_capacity_quota.py

```diff
@@ -2670,3 +2670,109 @@ def test_43958(self):
         assert resp, "Put object failed"
 
         self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-43857')
+    def test_43857(self):
+        """
+        Verify overwrite scenario for 1 object with check_on_raw=True
+        """
+        test_case_name = cortxlogging.get_frame()
+        test_cfg = self.csm_conf["test_43857"]
+        enabled = test_cfg["enabled"]
+        max_size = test_cfg["max_size"]
+        max_objects_1 = test_cfg["max_objects_1"]
+        max_objects_2 = test_cfg["max_objects_2"]
+        self.log.info("Step 1: Set max_size and max_objects quota for user")
+        payload = self.csm_obj.iam_user_quota_payload(enabled, max_size, max_objects_1,
+                                                       check_on_raw=True)
+        result, resp = self.csm_obj.verify_get_set_user_quota(self.user_id, payload,
+                                                              verify_response=True)
+        assert result, resp
+        self.log.info("Step 2: Try uploading 1 object(obj-1) of max_size: %s ", max_size)
```

Comment:
this will be successful, so can we remove "Try" from this ?

Reply:
you mean from msg?

Reply:
done